### DISCUSSION TITLE
Can i enroll in GitHub

### DISCUSSION THREAD

**Discussion by adnan-qazii** [community/community#102695] [GitHub Education]
### Hi everyone,

Any suggestions on it how to get it ?

**Comment by darrehanrasool** [comment]
Yep you can enroll

**Comment by ettaboyle** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
